icu4c/source/common/dictbe.h

```diff
@@ -362,10 +369,19 @@ class CjkBreakEngine : public DictionaryBreakEngine {
      * @internal
      */
   UnicodeSet                fHangulWordSet;
+  UnicodeSet                fHiraganaWordSet;
```

Comment:
As I mention elsewhere, since fHiraganaWordSet and fParticleSet are used in the same place, you should merge them into one Hashtable fSkipSet and load both content into it.